test/e2e/run_volume_test.go

```diff
@@ -665,6 +665,69 @@ VOLUME /test/`, ALPINE)
 		Expect(found).Should(BeTrue())
 	})
 
+	It("podman run with --mount and U flag", func() {
+		u, err := user.Current()
+		Expect(err).To(BeNil())
+		name := u.Username
+		if name == "root" {
+			name = "containers"
+		}
+
+		content, err := ioutil.ReadFile("/etc/subuid")
+		if err != nil {
+			Skip("cannot read /etc/subuid")
+		}
+
+		if !strings.Contains(string(content), name) {
+			Skip("cannot find mappings for the current user")
+		}
+
+		mountPath := filepath.Join(podmanTest.TempDir, "foo")
+		os.Mkdir(mountPath, 0755)
+
+		// false bind mount
+		vol := "type=bind,src=" + mountPath + ",dst=" + dest + ",U=false"
+		session := podmanTest.Podman([]string{"run", "--rm", "--user", "888:888", "--mount", vol, ALPINE, "stat", "-c", "%u:%g", dest})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+		found, _ := session.GrepString("888:888")
+		Expect(found).Should(BeFalse())
```

Comment:
Please use `Expect(session.OutputToString()).ShouldNot(Equal("888:888"))` instead. The same for the other tests below, just change `ShouldNot` to `Should` depending if this is true or false.

Reply:
done